test/Datadog.Trace.ClrProfiler.Managed.Tests/ScopeDBFactoryTests.cs

```diff
@@ -0,0 +1,148 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Data;
+using System.Data.Common;
+using Datadog.Trace.Configuration;
+using MySql.Data.MySqlClient;
+using Npgsql;
+using Xunit;
+
+namespace Datadog.Trace.ClrProfiler.Managed.Tests
+{
+    public class ScopeDBFactoryTests
+    {
+        public static IEnumerable<object[]> GetDbCommandScopeData()
+        {
+            yield return new object[] { new System.Data.SqlClient.SqlCommand() };
+            yield return new object[] { new MySqlCommand() };
+            yield return new object[] { new NpgsqlCommand() };
+#if !NET452
+            yield return new object[] { new Microsoft.Data.SqlClient.SqlCommand() };
+#endif
+        }
+
+        [Theory]
+        [MemberData(nameof(GetDbCommandScopeData))]
+        public void CreateDbCommandScope_ReturnsNullForExcludedAdoNetTypes(IDbCommand command)
+        {
+            // Set up tracer
+            var collection = new NameValueCollection
+            {
+                { ConfigurationKeys.AdoNetExcludedTypes, command.GetType().FullName }
+            };
+            IConfigurationSource source = new NameValueConfigurationSource(collection);
+            var tracerSettings = new TracerSettings(source);
```

Comment:
FYI, `TracerSettings.AdoNetExcludedTypes` is mutable so in this case you could have used:
```csharp
var tracerSettings = new TracerSettings();
settings.AdoNetExcludedTypes.Add(command.GetType().FullName);
```

Last time we talked about this it was a get-only property.